x-pack/plugins/secops/public/components/notes/note_card/index.test.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+
+import { NoteCard } from '.';
+
+describe('NoteCard', () => {
+  const created = new Date();
+  const rawNote = 'noteworthy';
+  const user = 'elastic';
+
+  test('it renders a note card header', () => {
+    const wrapper = mountWithIntl(<NoteCard created={created} rawNote={rawNote} user={user} />);
+
+    expect(wrapper.find('[data-test-subj="note-card-header"]').exists()).toEqual(true);
+  });
+
+  test('it renders a note card body', () => {
+    const wrapper = mountWithIntl(<NoteCard created={created} rawNote={rawNote} user={user} />);
+
+    expect(wrapper.find('[data-test-subj="note-card-body"]').exists()).toEqual(true);
```

Comment:
Isn't this the same thing as just passing the selector to exists as in:
expect(wrapper.exists('[data-test-subj="note-card-body"]')).toEqual(true);

Ref: https://airbnb.io/enzyme/docs/api/ShallowWrapper/exists.html

Reply:
@FrankHassanabad per our out-of-band conversation, the `enzyme` doc linked in the comment above suggests we may use `wrapper.exists` per your suggestion, however:

```
wrapper.exists() // no parameters, type checks ok
```
...will pass a type check, but

```
wrapper.exists('this is a selector and should compile') // fails to type check.
```
...fails to type check.

Again, per our out-of-band conversation, since you're able to reproduce this type check error, it's not possible to implement the suggested change at this time.